x-pack/plugins/reporting/server/lib/tasks/execute_report.test.ts

```diff
@@ -91,7 +84,18 @@ describe('Execute Report Task', () => {
   });
 
   it('throws during reporting if Kibana starts shutting down', async () => {
-    const encryptedHeaders = await encryptHeaders({});
+    mockReporting.getExportTypesRegistry().register({
+      id: 'noop',
+      name: 'Noop',
+      setup: jest.fn(),
+      start: jest.fn(),
+      getSpaceId: jest.fn(),
+      createJob: () => new Promise(() => {}),
+      runTask: () => new Promise(() => {}),
+      jobContentExtension: 'pdf',
+      jobType: 'noop',
+      validLicenses: [],
+    });
```

Comment:
This test needs to declare in "inline" export type and register it, same as the code in`main`.

The "task runner" that is created for these tests is a type of object that will talks to Task Manager, so there aren't really any changes we need to do for this refactoring :)

Reply:
Oho!! That's good to know - I definitely was trigger happy getting rid of task runner :sweat_smile: